server/controllers/inventory/reports/changes.js

```diff
@@ -44,6 +46,11 @@ async function inventoryChanges(req, res, next) {
   const params = _.clone(req.query);
   const metadata = _.clone(req.session);
 
+  const enterpriseId = req.session.enterprise.id;
+  const { currencyId } = params;
+  const exchangeRate = await Exchange.getExchangeRate(enterpriseId, currencyId, new Date());
```

Comment:
I don't think it is appropriate to have an exchange rate in this report, unless you are going to exchange the values with the rate of the _date the value was changed_.  We do this on the account statement report:

![image](https://user-images.githubusercontent.com/896472/146896867-9f1eb936-75e2-48fe-8019-60f6b4a7aa0d.png)

We then have to mention the rate used for each date.  I'm not sure if it is worth having this feature though.   What do you think?

----

Alternatively, we could just compute the exchanged value when we store the change.  That would mean that anytime we change the value, we would have an exchange value computed to display.  The problem is that if we go back and change the exchange rate, this value would get out of sync with the real exchange rate.

Reply:
Ok.  For now, I've remove multi-currency support.